addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -1014,10 +1014,6 @@ var SnippetEditor = Widget.extend({
         // If the helper is inside the iframe, we want pointer events on the
         // frame element so that they reach the window and properly apply
         // the cursor.
```

Comment:
While this comment is still true, do we still want it now that its related lines of code are removed ?
Given that those 7 lines are still inside `drag_and_drop.js`'s `onDragStart` (and fulfilling their purpose for Firefox), I believe this comment can be removed.